selfdrive/car/honda/carstate.py

```diff
@@ -105,23 +105,15 @@ def get_can_signals(CP, gearbox_msg, main_on_sig_msg):
     else:
       checks += [("CRUISE_PARAMS", 50)]
 
-  if CP.carFingerprint in (CAR.ACCORD, CAR.ACCORDH, CAR.CIVIC_BOSCH, CAR.CIVIC_BOSCH_DIESEL, CAR.CRV_HYBRID, CAR.INSIGHT, CAR.ACURA_RDX_3G, CAR.HONDA_E):
-    signals += [("DRIVERS_DOOR_OPEN", "SCM_FEEDBACK", 1)]
-  elif CP.carFingerprint == CAR.ODYSSEY_CHN:
-    signals += [("DRIVERS_DOOR_OPEN", "SCM_BUTTONS", 1)]
-  elif CP.carFingerprint == CAR.HRV:
-    signals += [("DRIVERS_DOOR_OPEN", "SCM_BUTTONS", 1),
-                ("WHEELS_MOVING", "STANDSTILL", 1)]
-  else:
-    signals += [("DOOR_OPEN_FL", "DOORS_STATUS", 1),
-                ("DOOR_OPEN_FR", "DOORS_STATUS", 1),
-                ("DOOR_OPEN_RL", "DOORS_STATUS", 1),
-                ("DOOR_OPEN_RR", "DOORS_STATUS", 1),
-                ("WHEELS_MOVING", "STANDSTILL", 1)]
-    checks += [
-      ("DOORS_STATUS", 3),
-      ("STANDSTILL", 50),
-    ]
+  signals.extend((signal, door_sig_msg, 1) for signal in door_signals)
+
+  if CP.carFingerprint not in HONDA_NO_WHEELS_MOVING_SIGNAL:
+    signals += [("WHEELS_MOVING", "STANDSTILL", 1)]
+    if CP.carFingerprint != CAR.HRV:
+      checks += [
+        ("DOORS_STATUS", 3),
+        ("STANDSTILL", 50),
+      ]
```

Comment:
I *think* there should not be an HRV exception for the STANDSTILL check.

And we should have separate sets for cars without WHEELS_MOVING and without DOORS_STATUS.

But anyway I left the original logic intact for now.

Reply:
certainly HRV hits this code path, so it should not have the STANDSTILL check exception
```python
ret.standstill = not cp.vl["STANDSTILL"]["WHEELS_MOVING"]
```

HRV only happens to work because STANDSTILL check is added again later:
```python
  if CP.openpilotLongitudinalControl:
    ...
    checks += [("STANDSTILL", 50)]
```